app/models/bitbucket/api.rb

```diff
@@ -71,39 +95,46 @@ def self.parse_single_issue(response = "")
     issue = JSON.parse(response)
 
     linked_account = LinkedAccount::Bitbucket.find_or_create_from_api_response(
-      author_name: issue['reported_by'] && issue['reported_by']['username'],
-      author_image_url: issue['reported_by'] && issue['reported_by']['avatar']
+      author_name: issue['reporter'] && issue['reporter']['username'],
+      author_image_url: issue['reporter'] && issue['reporter']['links'] &&
+                        issue['reporter']['links']['avatar'] &&
+                        issue['reporter']['links']['avatar']['href']
     )
 
     {
-      number: issue['local_id'],
+      number: issue['id'],
       title: issue['title'],
       state: issue['status'],
       priority: issue['priority'],
-      body: issue['content'],
+      body: issue['content'] && issue['content']['html'],
       can_add_bounty: issue['status'] == 'new' || issue['status'] == 'open',
-      url: issue['resource_uri'].gsub('/1.0/repositories', 'https://bitbucket.org').gsub('/issue/', '/issues/'),
-      owner: issue['reported_by'] ? issue['reported_by']['username'] : '',
+      url: issue['links'] && issue['links']['html'] &&
+           issue['links']['html']['href'],
       author: linked_account,
-      remote_created_at: DateTime.parse(issue['utc_created_on']),
-      remote_updated_at: DateTime.parse(issue['utc_last_updated'])
+      owner: issue['assignee'] ? issue['assignee']['username'] : nil,
+      remote_created_at: DateTime.parse(issue['created_on']),
+      remote_updated_at: issue['updated_on'] &&
+                         DateTime.parse(issue['updated_on'])
     }
   end
 
-  def self.parse_issue_list(base_url, response = "")
+  def self.parse_issue_list(base_url, response = "", state = {})
     data = JSON.parse(response)
-    data['issues'].map do |issue|
+    state[:next_url] = data['next']
+    data['values'].map do |issue|
       {
-        number: issue['local_id'],
+        number: issue['id'],
         title: issue['title'],
         state: issue['status'],
```

Comment:
according to api v2, it should be 'state' instead of 'status'. Please modify issue's state for this as well as issue's can_add_bounty since it depends on this as well. Thank you

Reply:
Thank you. I missed it.